blueprints/app/files/package.json

```diff
@@ -15,7 +15,7 @@
     "lint:hbs": "ember-template-lint .",
     "lint:js": "eslint .",
     "start": "ember serve",
-    "test": "ember test"
+    "test": "<% if (yarn) { %>yarn lint:js<% } else { %>npm run lint:js<% } %> && ember test"
```

Comment:
I left off `npm run lint:hbs` because there is currently no way that I know to take template linting out of `ember test`. Once that is possible, we can add it here. @rwjblue correct me if I'm wrong.

Reply:
You can remove `ember-cli-template-lint` and replace with `ember-template-lint` directly. This will keep Test'em from linting the has files, but you can still `yarn lint:hbs` with no issues.